Plugins/Flow.Launcher.Plugin.Program/Settings.cs

```diff
@@ -13,7 +13,8 @@ public class Settings
 
         public bool EnableStartMenuSource { get; set; } = true;
 
-        public bool EnableDescription { get; set; } = true;
+        public bool EnableDescription { get; set; } = false;
+        public bool EnableHideLnkPath { get; set; } = true;
```

Comment:
feel like this variable should be `EnableHideAppPath` because you are hiding not only Lnk but also could be the file path to the app executable right?

Reply:
yeah. I'll chage it.